adagrams/game.py

```diff
@@ -1,13 +1,126 @@
 from random import randint
 
 def draw_letters():
-    pass
+    tiles = [
+    'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 
+    'B', 'B',
+    'C', 'C',
+    'D', 'D', 'D', 'D', 
+    'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 
+    'F', 'F', 
+    'G', 'G', 'G', 
+    'H', 'H',  
+    'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 
+    'J',  
+    'K', 
+    'L', 'L', 'L', 'L', 
+    'M', 'M', 
+    'N', 'N', 'N', 'N', 'N', 'N', 
+    'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 
+    'P', 'P', 
+    'Q', 
+    'R', 'R', 'R', 'R', 'R', 'R', 
+    'S', 'S', 'S', 'S', 
+    'T', 'T', 'T', 'T', 'T', 'T',
+    'U', 'U', 'U', 'U', 
+    'V', 'V', 
+    'W', 'W', 
+    'X', 
+    'Y', 'Y', 
+    'Z'
+    ]
+    selected_index = []
+    picked_letters = []
+    for i in range(0, 10):
```

Comment:
If I did not know how to play Adagrams, I would not understand the significance of 10. If I were a dev who tried to read this, I might be wondering "Why do we only loop 10 times"?

In Python, 10 on this line is what is known as a "Magic Number". We use the term to refer to hard coded values that just kind of appear without any context to them!

It would be more descriptive to use a constant variable to reference 10 so that your code is more self-documenting. 

```suggestion
    NUM_TILES_ALLOWED_IN_HAND = 10
    for _ in range(NUM_TILES_ALLOWED_IN_HAND):
```